http/requests.go

```diff
@@ -12,21 +12,37 @@ const (
 	OrgName = "org"
```

Comment:
remove this line, if it's not being used everywhere

Reply:
Fixed